NorthstarDLL/util/printmaps.cpp

```diff
@@ -142,6 +143,27 @@ int, __fastcall, (const char const* cmdname, const char const* partial, char com
 	return numMaps;
 }
 
+ADD_SQFUNC(
+	"array<string>",
+	NSGetLoadedMapNames,
+	"",
+	"Returns a string array of loaded map file names",
+	ScriptContext::UI | ScriptContext::CLIENT | ScriptContext::SERVER)
+{
+	// Maybe we should call this on mods reload instead
+	RefreshMapList();
```

Comment:
`RefreshMapList()` polls fs so it might be worth moving logic from the [`map` autocompletion func](https://github.com/R2Northstar/NorthstarLauncher/blob/7ee56405a5de3ce8759f6e4e64faadbaa7bdf1a0/NorthstarDLL/util/printmaps.cpp#L115-L122) directly into `RefreshMapList()`. Will look into this friday

Reply:
Addressed in 2bfd2de

Reply:
Did you test this? Asking since we've had problems reading time from the globals before.

Reply:
Tested it by logging when we're refreshing and just spamming the `map` command auto completion. Printed a message every 10 seconds like it is supposed to

Reply:
Okay, sounds good.